acolorbright/acb-tools-and-configs
# Update editor config and eslint style to use tabs for indentation

## Description
Changes `.editorconfig` to use tabs instead of spaces for indentation. Removes `root = true` so the display of the tabs will be determined by personal preference when you have a `.editorconfig` file in the root of your code folder; for instance `~/code/.editorconfig`.

Updated the `eslint` `indent` setting as well to allow tabs and explicitly set `allowIndentationTabs: true` to the [no-tabs](https://eslint.org/docs/rules/no-tabs) rule.

Also added `.prettierrc` file to `eslint-config`. 

## Context
Outcome of a discussion we had around using tabs/spaces for indentation is to use tabs and then use a personal `.editorconfig` file at a higher level that controls how you want to display tabs.

## How to test
Check out this branch locally. Create an `.editorconfig` at a higher level than the project folder with:

```
root = true

[*]
indent_size = 2
```

And confirm that tabs are displayed as 2 spaces. Also try changing the `indent_size` to a different number like 4 or 8.

## Documentation
N/A

## Screenshots / Screen recordings
N/A

## Requesting feedback on
N/A
